service/state/core_access_test.go

```diff
@@ -0,0 +1,31 @@
+package state
+
+/*
+The test below has been commented out for now as the mock celestia-app has issues
+initializing the chain, and therefore cannot be used properly in the test yet.
+*/
```

Comment:
I'd prefer a skipped test instead of commented out. Also an issue that describes the necessary steps to unblock the test.

Reply:
Deleted test, tracking [here](https://github.com/celestiaorg/celestia-node/issues/570)